src/Bicep.LangServer.IntegrationTests/Helpers/TextDocumentParamHelper.cs

```diff
@@ -11,7 +11,7 @@ namespace Bicep.LangServer.IntegrationTests.Helpers
     public static class TextDocumentParamHelper
     {
         public static DidOpenTextDocumentParams CreateDidOpenDocumentParams(DocumentUri documentUri, string text, int version) =>
-            new DidOpenTextDocumentParams
+            new()
```

Comment:
Very much a nitpick, but for parameter-less constructors, any way to fine tune to:
```csharp
new {
    ...
}
```
Instead of
```csharp
new() {
    ...
}
```

Reply:
I am not sure, are any of these style rules applicable for that? https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/#legend